.bazelci/presubmit.yml

```diff
@@ -223,6 +225,7 @@ tasks:
     platform: windows
     environment:
       RULES_RUST_CRATE_UNIVERSE_BOOTSTRAP: true
+      OPENSSL_STATIC: "1"
```

Comment:
Could this potentially be passed as an `env` attribute to the bootstrap definition to avoid having two places where environment variables are set?